app/cloud.go

```diff
@@ -200,3 +204,33 @@ func (a *App) SendCloudTrialEndWarningEmail(trialEndDate, siteURL string) *model
 
 	return nil
 }
+
+func (a *App) SendCloudTrialEndedEmail() *model.AppError {
+	sysAdmins, e := a.getSysAdminsEmailRecipients()
+	if e != nil {
+		return e
+	}
```

Comment:
`err` as naming convention

```suggestion
	sysAdmins, err := a.getSysAdminsEmailRecipients()
	if err != nil {
		return err
	}
```